### PR TITLE
add nested amp-accordion sample

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@kul3r4 PTAL

File: src/20_Components/amp-accordion.html [hunk 1]
```diff
@@ -74,5 +74,44 @@
...
+    
+    -->
+    <amp-accordion >
+      <section>
+        <h4>Section 1</h4>
+        <p>Bunch of content.</p>
+      </section>
+      <section>
+        <h4>Section 2</h4>
+        <amp-accordion>
```
Comment (kul3r4) [inline src/20_Components/amp-accordion.html]: would it be possible to have a different style (such as changing the font or the color) for the nested content? Just to be visually clearer

**Comment by sebastianbenz** [comment]
PTAL

**Comment by kul3r4** [comment]
LGTM

**Pull request merged**